src/legacy/server/saved_objects/saved_objects_mixin.test.js

```diff
@@ -18,6 +18,48 @@
  */
 
 import { savedObjectsMixin } from './saved_objects_mixin';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { mockKibanaMigrator } from '../../../core/server/saved_objects/migrations/kibana/kibana_migrator.mock';
```

Comment:
shouldn't we export it from `core/server/mocks`?

Reply:
I thought we only want to expose mocks that plugins are likely to depend on from `core/server/mocks`. Since this is an "internal" mock, we will probably not need to expose it outside of core when we remove legacy.